it/auth/IgnoreEmploymentExpensesAuthSupportingAgentsAllowedISpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Can you rename IgnoreEmploymentExpensesAuthSupportingAgentsAllowedISpec.scala to IndividualsExpensesIgnoreEmploymentExpensesAuthSupportingAgentsAllowedISpec . As we want the file name to have the API name in it.